### PR TITLE
Display timestamp while view ETW event

### PR DESCRIPTION
**Pull request by iaalm**
Addressing #1700 
As the `Time MSec` is quite useful for filter, I just adding one extra field.
Manually tested worked.

### CODE REVIEW

**Comment by iaalm** [comment]
@microsoft-github-policy-service agree

**Comment by iaalm** [comment]
@brianrob could you help review?

File: src/PerfView/EventViewer/EventWindow.xaml [hunk 1]
```diff
@@ -242,6 +242,13 @@
                         MinWidth="60"
                         IsReadOnly="True" />
                         <WPFToolKit:DataGridTextColumn
+                        x:Name="TimeStampColumn"
+                        Header="TimeStamp"
+                        Binding="{Binding TimeStamp, StringFormat=u}"
```
Comment (pharring) [inline src/PerfView/EventViewer/EventWindow.xaml:247]: StringFormat u (Universal) is compact, but it lacks precision (nearest second).
Consider `StringFormat=o` (ISO8601) which has more precision.

Comment (brianrob) [inline src/PerfView/EventViewer/EventWindow.xaml:247]: I would like to make sure that the column description contains the time zone - specifically either "UTC" or "Trace Local".  It looks like `u` is UTC, while `o` contains the time zone information.

**Review by brianrob — COMMENTED** [review]
Thanks @iaalm for contributing this.  A comment below - also it looks like @pharring has a comment as well.

File: src/PerfView/EventViewer/EventWindow.xaml (hunk 1 above)
Comment (iaalm) [reply src/PerfView/EventViewer/EventWindow.xaml]: Fixed

Comment (iaalm) [reply src/PerfView/EventViewer/EventWindow.xaml]: Sure, adding `Trace Local`

**Comment by pharring** [comment]
Time zones are always tricky. My experience with services tells me to convert everything to UTC. That tends to be a safe approach -- avoiding all question about time zone -- but, it's probably not appropriate for PerfView (expect, perhaps as an option).
With PerfView, there are two local time zones to consider: The time zone on the machine where the data was gathered (if known) and the time zone on the machine where PerfView is running. When we say "Trace local", I assume that means that the times will be displayed using the time zone on the machine where the trace was gathered (including any adjustment for daylight saving that was in effect when the trace started).
I seem to remember that ETW sessions include a time zone offset in the header. I don't know if netperf does.

I wanted to call this out, so we are clear and precise about what "Trace Local" means.

Oh, now that I think of it, please update the Help documentation to describe the new column. Add it to the section headed "Column Descriptions". You'll then get a chance to explain what a "Trace Local" timestamp means.

**Review by pharring — APPROVED** [review]
See my comment about being precise about time zones. Would be nice to have it precisely defined in the help doc.

**Comment by iaalm** [comment]
@pharring That's a good point. Seems these two time zones are both accessible, so I'm thinking about make an option for it. Maybe default using the time zone of gathered machine.

**Comment by brianrob** [comment]
> Time zones are always tricky. My experience with services tells me to convert everything to UTC. That tends to be a safe approach -- avoiding all question about time zone -- but, it's probably not appropriate for PerfView (expect, perhaps as an option). With PerfView, there are two local time zones to consider: The time zone on the machine where the data was gathered (if known) and the time zone on the machine where PerfView is running. When we say "Trace local", I assume that means that the times will be displayed using the time zone on the machine where the trace was gathered (including any adjustment for daylight saving that was in effect when the trace started). I seem to remember that ETW sessions include a time zone offset in the header. I don't know if netperf does.
> 
> I wanted to call this out, so we are clear and precise about what "Trace Local" means.
> 
> Oh, now that I think of it, we should update the Help documentation to describe the columns in the Event Viewer. There's a section for the Event Viewer, but it doesn't describe the various columns. (There is a "Column Description" section for the Stack Viewer, but not for the Event Viewer) We should add a "Column Description" section to the "The Event Viewer" section. We'll then get a chance to explain what "Time MSec" and "Timestamp (Trace Local)" mean.

Agreed with @pharring's comments here.  We should make sure to clarify time zone - I like the idea of a "Trace Local" option, but if that is too complicated, I am OK with UTC as the only option.  I just want to make sure that we make it clear which one the user is looking at.

Also agreed on updating the help and description.

**Comment by iaalm** [comment]
@brianrob Please help review again.

File: src/PerfView/EventViewer/EventWindow.xaml [hunk 2]
```diff
@@ -79,6 +79,7 @@
             <Menu DockPanel.Dock="Top" Background="{DynamicResource ControlDefaultBackground}">
                 <MenuItem Header="_File">
                     <MenuItem Header="_Parent Window" Click="DoOpenParent"/>
+                    <MenuItem Header="_Show Local Time" IsCheckable="True" IsChecked="False" Checked="DoUseLocalTime" Unchecked="DoUseOriginTime"/>
```
Comment (pharring) [inline src/PerfView/EventViewer/EventWindow.xaml:82]: Suggestion: Create a new "View" or "Options" menu for this. It's not a "File" thing.

**Review by pharring — APPROVED** [review]

**Comment by iaalm** [comment]
@brianrob  for review😀

**Comment by iaalm** [comment]
@brianrob gentle ping..

**Review by brianrob — APPROVED** [review]
LGTM.  Thanks @iaalm!

**Pull request merged**